internal/server/server.go

```diff
@@ -141,6 +142,12 @@ func (e RedirectError) To() string {
 	return string(e)
 }
 
+type RfcError struct {
```

Comment:
Would `ProblemError` be a better name, what RFC does it refer to?

Reply:
Yeah, that's clearer and updated now. The RFC in question is [7807](https://www.rfc-editor.org/rfc/rfc7807) FWIW